templates/ingress.yaml

```diff
@@ -16,19 +17,14 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
```

Comment:
I Still think we should keep this specific annotation directly here -- we've got a separate value for it, and this change makes that value otherwise unused.

Reply:
We may want to move this logic to Houston and remove this special config. Same as my earlier comment below.